index.cjs

```diff
@@ -119,10 +121,32 @@ function copyQuickstarts() {
     path.join(replicacheQuickstartsDest, 'package.json'),
     JSON.stringify(packageJson, null, 2) + os.EOL
   )
+
+}
+
+function createLicense() {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  rl.question(
```

Comment:
Big picture though - it seems like this will just create a license but you still have to add it to the env vars. So this just prevents you from typing `npx replicache create-license` manually. I think this is a bit awkward. It's not even clear at this point that the user knows what a license is.

I think this experience needs a bit more design. There are questions like:

- should we explain to the user a bit more about what a license is?
- should we store the license somewhere so future commands use it automatically?
- what if the user has multiple apps. we intent that each app has its own license key?

Let's not add this until these questions are answered otherwise I think the experience is overall worse.